pizjuce/MiddyMorphy/MidiMorphPlugInInterface.h

```diff
@@ -3,14 +3,6 @@
 #include "../_common/PizAudioProcessor.h"
```

Comment:
Ugh, I'll look into it as I think we're on the same page that at least in the headers it's not so helpful to add the namespaces.

Reply:
I removed this one and as expected a lot of other files needed to be adapted.
To make it worse therefore I illogically decided to get rid of the "using namspace" in all files within the pizjuce dir.

Reply:
I appreciate your rigor :-)

Still, it will most likely be next week until I get this reviewed, since I will be quite busy otherwise for the next days...

Reply:
Thanks and no rush! I hope we can get this one to a point where it's acceptable for both of us..
This one got a little bit out of hand.. should've done this in separate chunks/PRs, I guess :(